File: tests/programs/Example_PHP/bin/ExampleSimple.php
<?php

require_once('Tinkerforge/IPConnection.php');
require_once('Tinkerforge/BrickletDualRelay.php');

use Tinkerforge\IPConnection;
use Tinkerforge\BrickletDualRelay;

const HOST = 'localhost';
const PORT = 4223;
const UID = 'xyz'; // Change to your UID

$ipcon = new IPConnection(); // Create IP connection
$dr = new BrickletDualRelay(UID, $ipcon); // Create device object

$ipcon->connect(HOST, PORT); // Connect to brickd
// Don't use device before ipcon is connected

// Turn relays alternating on/off for 10 times with 1 second delay
for($i = 0; $i < 10; $i++) {
    sleep(1);

    if ($i % 2 == 1) {
        $dr->setState(TRUE, FALSE);
    } else {
        $dr->setState(FALSE, TRUE);
    }
}

$ipcon->disconnect();

?>

File: tests/programs/Example_VB.NET/bin/ExampleSimple.vb
Imports Tinkerforge

Module ExampleSimple
    Const HOST As String = "localhost"
    Const PORT As Integer = 4223
    Const UID As String = "xyz" ' Change to your UID

    Sub Main()
        Dim ipcon As New IPConnection() ' Create IP connection
        Dim dr As New BrickletDualRelay(UID, ipcon) ' Create device object

        ipcon.Connect(HOST, PORT) ' Connect to brickd
        ' Don't use device before ipcon is connected

        ' Turn relays alternating on/off for 10 times with 1 second delay
        Dim i As Integer

        For i = 1 To 10
            System.Threading.Thread.Sleep(1000)

            If i Mod 2 = 0 Then
                dr.SetState(True, False)
            Else
                dr.SetState(False, True)
            End If
        Next i

        ipcon.Disconnect()
    End Sub
End Module
